python/dev.py

```diff
@@ -255,6 +260,12 @@ def main() -> None:
         action="store_true",
         help="Install Python dependencies without cache",
     )
+    build_parser.add_argument(
+        "--glide-version",
```

Comment:
the version is also relevant for the async client, you can set it in `build_async_client` when we execute 'maturin develop'